toolkit/tools/imagecreator/main.go

```diff
@@ -28,7 +28,7 @@ type ImageCreatorCmd struct {
 	OutputImageFile   string   `name:"output-image-file" help:"Path to write the customized image to."`
 	OutputImageFormat string   `name:"output-image-format" placeholder:"(vhd|vhd-fixed|vhdx|qcow2|raw)" help:"Format of output image." enum:"${imageformat}" default:""`
 	Distro            string   `name:"distro" help:"Target distribution for the image." enum:"azurelinux,fedora" default:"azurelinux"`
```

Comment:
Do we ever expect a config file to be usable by more than 1 distro and/or version? If not, then it would probably make sense to remove the `distro` and `distro-version` and add equivalent fields into the config file instead.